README.md

```diff
@@ -69,27 +70,270 @@ class Genre {
-just `String`s:
+We can then create instances for both of these classes and _pass in the
+instances_ rather than just `String`s when creating a `Book` instance:
 
 ```js
-let denisJohnson = new Author('Denis', 'Johnson');
-let historicalFiction = new Genre('Historical Fiction');
-
-let trainDreams2 = new Book(
-	'Train Dreams',
-	denisJohnson,
-	historicalFiction,
-	'2002'
+let tolkien = new Author('John Ronald Reuel', 'Tolkien');
+let fantasy = new Genre('Fantasy');
+
+let lordOfTheRings = new Book(
+	'The Lord of the Rings',
+	tolkien,
+	fantasy,
+	'1954'
+);
+```
+
+Our new `Book` instance now contains `Genre` and `Author` instances assigned as
+properties:
+
+```js
+lordOfTheRings;
+// => Book {
+//    title: 'The Lord of the Rings',
+//    author: Author { firstName: 'John Ronald Reuel', lastName: 'Tolkien' },
+//    genre: Genre { name: 'Fantasy' },
+//    publishingDate: '1954' }
+```
+
+Now that we've got a set of related class instances, we can reuse them as
+needed:
+
+```js
+let hobbit = new Book(
+	'The Hobbit, or There and Back Again',
+	tolkien,
+	fantasy,
+	'1937'
+);
+
+let silmarillion = new Book('The Silmarillion', tolkien, fantasy, '1977');
+```
+
+Using these instances, we've eliminated the need to repeatedly write `String`
+values, reducing the chance for errors and keeping our data consistent. Because
+the `tolkien` and `fantasy` values are pointing to instances of `Author` and
+`Genre`, if we change the instance, all references will update as well:
+
+```js
+tolkien.firstName = 'J.R.R.';
+lordOfTheRings;
+// => Book {
+//    title: 'The Lord of the Rings',
+//    author: Author { firstName: 'J.R.R.', lastName: 'Tolkien' },
+//    genre: Genre { name: 'Fantasy' },
+//    publishingDate: '1954' }
+lordOfTheRings.author.firstName;
+// => 'J.R.R.'
+```
+
+Here, `tolkien` and `lordOfTheRings.author` refer to the same instance of
+`Author`.
+
+As we see with `lordOfTheRings.author.firstName`, by storing an instance of the
+`Author` class as a property in `Book`, we gain access to properties of the
+`Author` class:
+
+```js
+lordOfTheRings.author; // => Author { firstName: 'J.R.R.', lastName: 'Tolkien' }
+lordOfTheRings.author.firstName; // => 'J.R.R.'
+lordOfTheRings.author.lastName; // => 'Tolkien'
+```
+
+We can even write methods in one class that use the properties of another:
+
+```js
+class Book {
+	constructor(title, author, genre, publishingDate) {
+		this.title = title;
+		this.author = author;
+		this.genre = genre;
+		this.publishingDate = publishingDate;
+	}
+
+	get caption() {
+		let title = this.title;
+		let first = this.author.firstName;
+		let last = this.author.lastName;
+		let date = this.publishingDate;
+		return `${title} was written by ${first} ${last} and published in ${date}.`;
+	}
+}
+```
+
+Here, we've created a `caption` getter that uses some properties of `Book`
+and some of `Author`. This is the most basic way classes can work together - by
+depending on each other for related data.
```

Comment:
I really like how this is broken down better now since it's a large chunk of code. I'm wondering if it might be useful to give a quick 1 sentence review of what `get` was, or to link back out to that lesson or a resource just in case students have already forgotten about its relevance.